### PR TITLE
data:Bharath-V26

### PR DESCRIPTION
**Pull request by Bharath-V26**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3177"></a>

### CODE REVIEW

**Comment by Bharath-V26** [comment]
This is the first file I'm creating

File: data/Bharath-V26.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "name": "Bharath V",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "I am an enthusiastic cloud-computing job seeker eager to utilize my technical and communication skills to help organization reach its goals.",
+  "avatar": "https://github.com/Bharath_V26.png",
```
Comment (Aadarsh805) [inline data/Bharath-V26.json:6]: the avatar URL should be `https://github.com/Bharath-V26.png`

**Review by Aadarsh805 — CHANGES_REQUESTED** [review]

**Review by EmmaDawsonDev — DISMISSED** [review]
This is almost ready to go, just need to correct your avatar link. Let us know if you need any help.

File: data/Bharath-V26.json (hunk 1 above)
Comment (eddiejaoude) [inline data/Bharath-V26.json:6]:
```suggestion
  "avatar": "https://github.com/Bharath-V26.png",
```

**Review by SaraJaoude — APPROVED** [review]
looks good

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Pull request merged**